admin/src/components/VendorTable.tsx

```diff
@@ -3,12 +3,12 @@ import { Button, Table } from "react-bootstrap";
 import { Link } from "react-router-dom";
 import "bootstrap/dist/css/bootstrap.min.css";
 import "./login.css";
-import VendorDetails from "../pages/VendorDetails";
 import { fetchVendors, deleteVendorById } from "./Service";
 import DeleteEntity from "./DeleteEntity";
+import { VendorInterface } from "../vendor/Vendor";
```

Comment:
Uh, na még két dolog:
- itt is elég Vendornak hívni
- Lehetne default export és akkor nem kell itt { }